mcs/resource_mcs_db_database.go

```diff
@@ -55,17 +70,37 @@ func resourceDatabaseDatabaseCreate(d *schema.ResourceData, meta interface{}) er
 	}
 
 	databaseName := d.Get("name").(string)
-	instanceID := d.Get("instance_id").(string)
+	dbmsIDRaw, dbmsIDOk := d.GetOk("dbms_id")
+	instanceIDRaw, instanceIDOk := d.GetOk("instance_id")
+	if !dbmsIDOk && !instanceIDOk {
+		return fmt.Errorf("one of instance_id or dbms_id must be set")
```

Comment:
lets mention dbms_id only, as instance_id is deprecated

Reply:
fixed